src/fragmentarium/ui/fragment/Download.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from 'react'
 import { DropdownButton, ButtonGroup, Dropdown } from 'react-bootstrap'
 import _ from 'lodash'
 import { Fragment } from 'fragmentarium/domain/fragment'
+import * as TeiExport from './TeiExport.js'
```

Comment:
All code should be TypeScript. JavaScript is no longer allowed in master.

Reply:
Changed, will be .tsx in next commit.

Reply:
word in function getWord(word:any,escaped_word_value:string) does not appear to be of class word but of instance of object therefore using type "any"

Reply:
Using `any` should be avoided as it will hide errors and creates more warnings (Adding types is an ongoing process, so this is only a warning and not an error.). Contents of a line should be of type [Token](https://github.com/ElectronicBabylonianLiterature/ebl-frontend/blob/master/src/transliteration/domain/token.ts), not everything on a line is a `Word`. Token types are not classes, so they are plain `objects`, but should have the properties matching the interface.

Reply:
> Changed, will be .tsx in next commit.

The file should be `.ts`.  `.ts` is plain TypeScript. `.tsx` must be used only if the file contaings JSX.

Reply:
Changed.